element_data_loader/path_utils.py

```diff
@@ -0,0 +1,57 @@
+import datajoint as dj
+import pathlib
+
+
+dj.config['enable_python_native_blobs'] = True
+
+
+def find_full_path(root_directories, relative_path):
```

Comment:
Why are these functions here? Who uses them?

Reply:
These functions are used in `element-calcium-imaging`, `element-array-ephys`, and `element-miniscope`.  So I have refactored the code into this repository to reduce duplicated code.